target/linux/lantiq/image/Makefile

```diff
@@ -267,9 +293,12 @@ endef
 TARGET_DEVICES += arcadyan_arv7525pw
 
 define Device/arcadyan_arv4519pw
+  DEVICE_VENDOR := Arcadyan
+  DEVICE_MODEL := ARV4519PW
+  DEVICE_ALT0_VENDOR := Vodafone
+  DEVICE_ALT0_MODEL := Pirelli
```

Comment:
Pirelli is yet another vendor, I'm having a hard time finding out what the product names were...

Reply:
For this device I think it would be acceptable to just remove the Vodafone/Pirelli.
As long as we cannot find a model name, the vendor won't help much.

Reply:
https://github.com/openwrt/openwrt/commit/3cbfc74cde8e7064b2b8c2cbb7aa20374e759dc0#diff-e50949faf4af26ef1981550b8495e7eeR6

```
  DEVICE_ALT0_VENDOR := Vodafone
  DEVICE_ALT0_MODEL := NetFasteR IAD 2
  DEVICE_ALT0_VENDOR := Pirelli
  DEVICE_ALT0_MODEL := P.RG A4201G
```

Reply:
Nice one ...